package.json

```diff
@@ -8,7 +8,7 @@
   "license": "MIT",
   "dependencies": {
     "@quintype/backend": "^1.23.2",
-    "@quintype/components": "^2.23.0-youtube-render.3",
+    "@quintype/components": "^2.25.3",
```

Comment:
how did you changed this ? package-lock.js file should also change if you updating any version .

Reply:
Created a branch and changed it in package-lock.json and installed it for the change to reflect in package-lock.json. Did not touch any js file